libraries/command_generators.rb

```diff
@@ -77,8 +78,16 @@ def etcdctl_bin
       '/usr/bin/etcdctl'
     end
 
+    def etcdctl_opts
+      if etcdctl_options
+        " #{etcdctl_options}"
+      else
+        ""
+      end
+    end
+
     def etcdctl_command
-      "#{etcdctl_bin} set #{etcd_prefix} '#{configuration.to_json}'"
+      "#{etcdctl_bin}#{etcdctl_opts} set #{etcd_prefix} '#{configuration.to_json}'"
```

Comment:
does having two spaces when `etcdctl_options` is nil break the configuration?  can lines 81-88 be removed?